packages/react-reconciler/src/__tests__/ReactDoubleInvokeEvents-test.internal.js

```diff
@@ -0,0 +1,500 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let React;
+let ReactFeatureFlags;
+let ReactNoop;
+let Scheduler;
+
+describe('ReactDoubleInvokeEvents', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    React = require('react');
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactNoop = require('react-noop-renderer');
+    Scheduler = require('scheduler');
+    ReactFeatureFlags.enableDoubleInvokingEffects = __VARIANT__;
```

Comment:
Yeah, I originally tried to do what you suggested but there were a ton of tests that would need to be updated and it didn't seem worth it since we're not sure this is the final version yet.